recipes/bpemb/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "bpemb" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd6ba916560ab1c79df2699f7a3002091a31957e2661a770cb92ac8febfc06ca
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
```

Comment:
not even osx?

Reply:
a dependency for it only works on linux: https://github.com/conda-forge/sentencepiece-feedstock/blob/master/recipe/meta.yaml